cli/providers/aws/route53/cnames.go

```diff
@@ -0,0 +1,127 @@
+package route53
+
+import (
+	"fmt"
+
+	awsSDK "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/route53"
+	"github.com/pchico83/i2kit/cli/providers/aws"
+	"github.com/pchico83/i2kit/cli/providers/aws/cf"
+	"github.com/pchico83/i2kit/cli/schemas/environment"
+	"github.com/pchico83/i2kit/cli/schemas/service"
+)
+
+//Create creates a CNAME resolving to a service
+func Create(s *service.Service, e *environment.Environment) error {
+	config, err := aws.GetConfig(e.Provider)
+	if err != nil {
+		return err
+	}
+	stack, err := cf.Get(s.Name, config)
```

Comment:
would the environment name matter here?

Reply:
Not yet, that full change would go in a different PR